pages/getinvolved.md

```diff
@@ -8,7 +8,14 @@ Below are some ways that you can get involved with Apache Fluo:
 
 ### Mailing list
 
-Join our public mailing list by sending an email to ```dev-subscribe@fluo.incubator.apache.org```. Discussions regarding issues and pull requests should remain on our GitHub pages ([Fluo][f], [Fluo Recipes][r], [Fluo website][w]). Our email list is for announcing releases, organizing meet ups, or any discussion that does not make sense for GitHub. 
+The `dev` lists is for general disscussion, announcing releases, organizing meet ups, etc.  For
+discussion regarding specific issues or pull request consider using Github ([Fluo][f], [Fluo
+Recipes][r], [Fluo website][w]).  The `notifications` and `commit` list are mailed to by automated
```

Comment:
Missing s in commits.

Could be "`commits` mailing lists receive email from automated services"